tests/cases/fourslash/quickInfoJsDocGetterSetter.ts

```diff
@@ -0,0 +1,68 @@
+/// <reference path='fourslash.ts' />
+
+//// class A {
+////     /**
+////      * getter A
+////      * @returns return A
+////      */
+////     get /*1*/x(): string {
+////         return "";
+////     }
+////     /**
+////      * setter A
+////      * @param value foo A
+////      * @todo empty jsdoc
+////      */
+////     set /*2*/x(value) { }
+//// }
+//// // override both getter and setter
+//// class B extends A {
+////     /**
+////      * getter B
+////      * @returns return B
+////      */
+////     get /*3*/x(): string {
+////         return "";
+////     }
+////     /**
+////      * setter B
+////      * @param value foo B
+////      */
+////     set /*4*/x(vale) { }
+//// }
+//// // not override
+//// class C extends A { }
+//// // only override setter
+//// class D extends A {
+////     /**
+////      * setter D
+////      * @param value foo D
+////      */
+////     set /*5*/x(val: string) { }
+//// }
+//// new A()./*6*/x = "1";
+//// new B()./*7*/x = "1";
+//// new C()./*8*/x = "1";
+//// new D()./*9*/x = "1";
+
+verify.quickInfoAt("1", "(getter) A.x: string", 'getter A', [{ name: "returns", text: "return A" }]);

```

Comment:
Also, I'd like to see a test of inherited jsdoc for other symbols like properties and methods.
It would be especially good to test inheritance from multiple bases, and inheritance from symbols that have multiple declarations. (I'm not sure what the latter would be -- maybe a type and namespace and property that all share the same name?)

Reply:
property is test here `quickInfoOnParameterProperties.ts` (its baseline file is updated in this PR)
method is test here `quickInfoJsDocTags5.ts`

So I guess there is no need to test the simple case?

As for the complex case, is it OK to add (There are same name methods and property in all interface and class, the property might be function or not)
- a class extends from base class and implement an interface
- a class implement two interfaces, then declare one namespace with all things with same name
Is this enough?

Reply:
Ooops, it is an error to have static method and namespace exported method with same name at the same time.
Then I guess namespace could not be used as a source of multiple declarations? I do not know much about namespace, or is there some method?

Reply:
I couldn't think of a way to merge a namespace and a class either. So it's probably fine to skip that case.